### PR TITLE
👹🤖 Graphical update on Unity and Gitlab.

### PR DESCRIPTION
**Pull request by Neuxz**
Adopt graphical  changes from [Mythen96](https://github.com/Mythen96) to Unity.

### CODE REVIEW

File: providers/gitlab.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+// gitlab.js
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/master/doc/user/project/integrations/webhooks.md
+// ========
+module.exports = {
+    parse: function (req, discordPayload) {
+        var body = req.body;
+        var username = body.user_name;
+        var type = body.object_kind;
+        var ref = body.ref;
+        discordPayload.username = username + " Commit";
```
Comment (Mythen96) [inline providers/gitlab.js]: As it seems that Jawnnypoo doesnt want to override the username and avatar of the webhook (he removed even my overrides for it) you should consider removing these lines.

File: providers/gitlab.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                    commits = commits + commit.author.name + " - ("+ "[`" + commit.id.substring(0, 7) + "`](" + commit.url + ") "+ ") [" + message.replace(/\n/g, ' ') + "](" + project.url + ") " + "\n";
+                }
+
+                discordPayload.embeds = [{
+                    title: "[" + project.name + ":" + project.branch + "] " + project.commits.length + " commit" + ((project.commits.length > 1) ? "s" : ""),
+                    url: project.url,
+                    author: user,
+                    description: commits,
+                    footer: {
+                        text: "",
```
Comment (Mythen96) [inline providers/gitlab.js]: Does it have any reasons why you removed the footer text?

File: providers/gitlab.js (hunk 1 above)
Comment (Jawnnypoo) [inline providers/gitlab.js]: Yeah, thanks for pointing that out. Not too much a fan of overriding the webhook configured image as the person probably set it intentionally. I am a fan of this new format though, looks great!

File: providers/gitlab.js (hunk 2 above)
Comment (Neuxz) [inline providers/gitlab.js]: No, I was playing around with it and did not set it to an value.

File: providers/gitlab.js (hunk 1 above)
Comment (Neuxz) [inline providers/gitlab.js]: Shure thing i change it back.

**Comment by Jawnnypoo** [comment]
This PR looks pretty strange, but I guess that is because you merged master into your branch before hand?

**Comment by Mythen96** [comment]
I think it looks strange because he is working in the glitch branch and not in the master branch. If you look closely he want to merge it also into the glitch branch and not into the master branch.

**Comment by Neuxz** [comment]
Whoops sorry

**Comment by Jawnnypoo** [comment]
Thanks!

**Comment by Jawnnypoo** [comment]
@Neuxz @Mythen96 If you guys are interested, I created a discord for skyhook so that we can potentially talk and collab on there

https://discord.gg/js7wD7p

**Pull request merged**